posts/2023-state-of-rust-analysis.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: 2022 Annual Rust Survey Results
+author: The Rust Survey Working Group in partnership with the Rust Foundation
+release: false
+---
+
+Hello, Rustaceans!
+
+For the 6th year in a row, the Rust Project conducted a survey on the Rust programming language, with participation from project maintainers, contributors, and those generally interested in the future of Rust. This edition of the annual State of Rust Survey opened for submissions on December 5 and ran until December 22, 2022. Thank you for your patience as we’ve worked to assemble this analysis of the survey responses. We plan to identify a more expedient and sustainable process going forward. 
```

Comment:
Is there a more detailed analysis of the data available? Specifically I was wondering whether there is a simple list of the questions + the aggregate results for each question on the github repo. If so, it'd be good to link to it.

Reply:
I don't think we have that now. While it's certainly possible, I think it might take a bit of work to get into a format that is legible for general consumption. I wouldn't block this (long overdue) post on that, but we should try our best to get something like that out ASAP (cc @nrc).